### PR TITLE
Removing split pickling

### PR DESCRIPTION
**Pull request by ArlindKadra**
Issue #431

### CODE REVIEW

File: openml/tasks/split.py [hunk 1]
```diff
@@ -125,11 +125,12 @@ def _from_arff_file(cls, filename, cache=True):
                             np.array(repetitions[repetition][fold][sample][0], dtype=np.int32),
                             np.array(repetitions[repetition][fold][sample][1], dtype=np.int32))
 
+            '''
```
Comment (janvanrijn) [inline openml/tasks/split.py]: PyCharm usually wants double quotes, right?

**Comment by ArlindKadra** [comment]
@mfeurer since you mentioned removing or fixing split pickling, I only commented out the code as a simple solution. I can remove it completely if you are ok with it. @janvanrijn we are using double quotes for docstrings.

**Comment by mfeurer** [comment]
I think removing it is fine. Reading those files isn't as costly as reading in the data.

**Comment by ArlindKadra** [comment]
@mfeurer sure, but we will not have a way to disable loading from cache.

**Comment by mfeurer** [comment]
I don't think we have this for anything else. As a split should never change I don't see a reason to keep this functionality.

**Pull request merged**